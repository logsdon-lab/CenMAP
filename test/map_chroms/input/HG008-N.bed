version https://git-lfs.github.com/spec/v1
oid sha256:8a00a7f0c141d4254703ee60249ae678ed0b659c33886484989231fdc7556918
size 285450
