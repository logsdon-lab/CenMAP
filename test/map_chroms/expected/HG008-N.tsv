version https://git-lfs.github.com/spec/v1
oid sha256:63f5af10730db70530627a01fbac194620e32d0a1a2855d70373e0e6dc078b5c
size 2073
